requirements.txt

```diff
@@ -1,4 +1,8 @@
 git+git://github.com/bigscience-workshop/promptsource@v0.1.0
 accelerate
 transformers
+datasets
```

Comment:
```suggestion
datasets
jinja2
```

Reply:
cc @Guitaricet

Reply:
Looks good. I think the new `requirements.txt` covers all of the dependencies issues I had.